yaml/pois.yaml

```diff
@@ -1139,6 +1156,17 @@ filters:
     output:
       <<: *output_properties
       kind: motorway_junction
+  # obelisk - note that this takes precedence over monument and memorial!
+  - filter: {man_made: obelisk}
+    min_zoom: { clamp: { min: 15, max: 17, value: { sum: [ { col: zoom }, 2.24 ] } } }
```

Comment:
For the Washington Monument in DC, if something is "tall", let's make it zoom 14. (greater than 20 meters?)

If it's medium tall then zoom 15? (greater than 10 meters?)

Reply:
Done in b90875b. Also pushed the priority of obelisks above artworks, as some appear to also have `tourism=artwork` on them, but obelisk seems more specific - especially for the >10m ones.